tools/SetupFlow/DevHome.SetupFlow/ViewModels/ReviewViewModel.cs

```diff
@@ -9,96 +9,118 @@
-            var appManagementTasksTotal = _setupFlowOrchestrator.GetTaskGroup<AppManagementTaskGroup>()?.SetupTasks.Count() ?? 0;
-            if (_setupFlowOrchestrator.IsSettingUpATargetMachine && repoConfigTasksTotal == 0 && appManagementTasksTotal == 0)
-            {
-                // either repo config or app management task group is required to setup target
-                return false;
-            }
-
-            return true;
-        }
-    }
-
-    public bool HasTasksToSetUp => Orchestrator.TaskGroups.Any(g => g.SetupTasks.Any());
-
-    public ReviewViewModel(
-        ISetupFlowStringResource stringResource,
-        SetupFlowOrchestrator orchestrator,
-        IHost host)
-        : base(stringResource, orchestrator)
-    {
-        _host = host;
-
-        NextPageButtonText = StringResource.GetLocalized(StringResourceKey.SetUpButton);
+using DevHome.SetupFlow.Models;
+using DevHome.SetupFlow.Services;
+using DevHome.SetupFlow.TaskGroups;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+
+namespace DevHome.SetupFlow.ViewModels;
+
+public partial class ReviewViewModel : SetupPageViewModelBase
+{
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(ReviewViewModel));
+
+    private readonly IHost _host;
+
+    private readonly SetupFlowOrchestrator _setupFlowOrchestrator;
+
+    [ObservableProperty]
+    private IList<ReviewTabViewModelBase> _reviewTabs;
+
+    [ObservableProperty]
+    private ReviewTabViewModelBase _selectedReviewTab;
+
+    [ObservableProperty]
+    private bool _readAndAgree;
+
+    [ObservableProperty]
+    [NotifyCanExecuteChangedFor(nameof(SetUpCommand))]
+    private bool _canSetUp;
+
+    [ObservableProperty]
+    private string _reviewPageTitle;
+
+    [ObservableProperty]
+    private string _reviewPageExpanderDescription;
+
+    [ObservableProperty]
+    private string _reviewPageDescription;
+
+    public bool HasApplicationsToInstall => Orchestrator.GetTaskGroup<AppManagementTaskGroup>()?.SetupTasks.Any() == true;
+
+    public bool RequiresTermsAgreement => HasApplicationsToInstall;
+
+    public bool HasMSStoreApplicationsToInstall
+    {
+        get
+        {
+            var hasMSStoreApps = Orchestrator.GetTaskGroup<AppManagementTaskGroup>()?.SetupTasks.Any(task =>
+            {
+                var installTask = task as InstallPackageTask;
+                return installTask?.IsFromMSStore == true;
+            });
+
+            return hasMSStoreApps == true;
+        }
+    }
+
+    public bool CanSetupTarget
+    {
+        get
+        {
+            var repoConfigTasksTotal = _setupFlowOrchestrator.GetTaskGroup<RepoConfigTaskGroup>()?.CloneTasks.Count ?? 0;
+            var appManagementTasksTotal = _setupFlowOrchestrator.GetTaskGroup<AppManagementTaskGroup>()?.SetupTasks.Count() ?? 0;
+            if (_setupFlowOrchestrator.IsSettingUpATargetMachine && repoConfigTasksTotal == 0 && appManagementTasksTotal == 0)
```

Comment:
Coding guidelines: "Use parentheses to make clauses in an expression apparent".
This is in Windows C++ guidelines and in [C# Common code conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#:~:text=Use%20parentheses%20to%20make%20clauses%20in%20an%20expression%20apparent).